.github/CODEOWNERS

```
@@ -2,7 +2,7 @@
 
 # https://github.com/apache/superset/issues/13351
 
-/superset/migrations/ @mistercrunch @michael-s-molina
+/superset/migrations/ @mistercrunch @michael-s-molina @geido @eschutho
```

Did you meant to add Diego?
```suggestion
/superset/migrations/ @mistercrunch @michael-s-molina @betodealmeida @eschutho
```